ComfyUI/server.py

```
@@ -625,6 +625,49 @@ async def get_queue(request):
             queue_info['queue_pending'] = current_queue[1]
             return web.json_response(queue_info)
 
+        @routes.get("/queue/progress")
+        async def get_queue_progress(request):
+            progress_info = {}
+            current_queue = self.prompt_queue.get_current_queue_volatile()
+            running_queue = current_queue[0]
+            pending_queue = current_queue[1]
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)